include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -6784,6 +6784,768 @@
       "capabilities" : [ "VariableLengthArrayINTEL" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpArbitraryFloatSinCosPiINTEL",
+      "class"  : "@exclude",
+      "opcode" : 5840,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "'A'" },
+        { "kind" : "LiteralInteger", "name" : "'M1'" },
+        { "kind" : "LiteralInteger", "name" : "'Mout'" },
+        { "kind" : "LiteralInteger", "name" : "'EnableSubnormals'" },
+        { "kind" : "LiteralInteger", "name" : "'RoundingMode'" },
+        { "kind" : "LiteralInteger", "name" : "'RoundingAccuracy'" }
+      ],
+      "capabilities" : [ "ArbitraryPrecisionFloatingPointINTEL" ],
+      "extensions" : [ "SPV_INTEL_arbitrary_precision_floating_point" ],
```

Comment:
I recommend using the "extensions" list only on the newly introduced capability enum;   the rest of the features are enabled by the capability only.
That removes doubt about how whether you need both the capability *and* the extension.

Reply:
Done, thanks!